src/drivers/Qt/HelpPages.cpp

```diff
@@ -99,15 +99,19 @@ void consoleWin_t::OpenHelpWindow(std::string subpage)
 
 	//printf("Looking for HelpFile '%s'\n", helpFileName.c_str() );
 
-#ifdef WIN32
+#if defined(WIN32)
 	// Windows specific HtmlHelp library function
-	helpWin = HtmlHelp( HWND(winId()), helpFileName.c_str(), HH_DISPLAY_TOPIC, (DWORD)NULL);
+#if defined(_UNICODE) || defined(UNICODE)
+    helpWin = HtmlHelp(HWND(winId()), helpFileName.wc_str(), HH_DISPLAY_TOPIC, (DWORD)NULL);
```

Comment:
What is this code for? I checked online docs and std::string does not have a wc_str method. Are you sure this compiles if UNICODE is true? I'm having trouble seeing what build issues you where having in this file to begin with.

Reply:
Fixed it, mistakes std::wstring for std::wc_str